CHANGELOG.md

```diff
@@ -3,6 +3,15 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v2.2.1 Romarin Curie - patch](https://github.com/nf-core/crisprseq/pull/174)
```

Comment:
Sorry I missed this link in the previous PR :) 
```suggestion
## [v2.2.1 Romarin Curie - patch](https://github.com/nf-core/crisprseq/releases/tag/2.2.1)
```

Also a reminder that we should add the date before merging this PR